src/cli/test/test_embedded_help.rb

```diff
@@ -11,63 +11,62 @@ def test_dir_glob
     Dir.chdir(File.join(File.dirname(__FILE__), '..'))
 
     # test things that should work in ruby or CLI
-    no_block_glob = Dir["*.txt", "*.rb"]
+    no_block_glob = Dir["*.txt", "*.cpp"]
     assert_instance_of(Array, no_block_glob)
     assert_includes(no_block_glob, 'CMakeLists.txt')
-    assert_includes(no_block_glob, 'embedded_help.rb')
-    puts no_block_glob.index('test_embedded_help.rb')
-    assert_nil(no_block_glob.index('test_embedded_help.rb'))
+    assert_includes(no_block_glob, 'main.cpp')
+    assert_nil(no_block_glob.index('test_main.cpp'))
```

Comment:
embedded_help.rb is no longer in the src/cli directory, but in the ruby/engine dir, so adjust the test. It must have been failing, **silently**, for a year and a half since https://github.com/NREL/OpenStudio/commit/50f6c0378ce1e5e4e48c525d7d1e2e80fd6adcb2